src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -536,23 +541,29 @@ private void handlePotentialStuckRepairs(LazyInitializer<Set<String>> busyHosts,
 
     return () -> {
       LOG.debug("getMetricsForHost {} / {} / {}", node, localDc, nodeDc);
-      try (JmxProxy nodeProxy
-          = context.jmxConnectionFactory.connect(node, context.config.getJmxConnectionTimeoutInSeconds())) {
-
-        NodeMetrics metrics = NodeMetrics.builder()
-            .withNode(node)
-            .withDatacenter(nodeDc)
-            .withCluster(nodeProxy.getClusterName())
-            .withPendingCompactions(nodeProxy.getPendingCompactions())
-            .withHasRepairRunning(nodeProxy.isRepairRunning())
-            .withActiveAnticompactions(0) // for future use
-            .build();
+      try (JmxProxy nodeProxy =
+          context.jmxConnectionFactory.connect(
```

Comment:
trivial: put the operator (`=`) on the new line. (so the precedence becomes: operators or arguments start a wrapped line)